api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -784,6 +795,9 @@ message HttpConnectionManager {
   // value.
   SetCurrentClientCertDetails set_current_client_cert_details = 17;
 
+  // The matcher for forwarding client cert details.
+  xds.type.matcher.v3.Matcher forward_client_cert_matcher = 60;
```

Comment:
This is API only PR. Could you add a `[#not-implemented-hide:]` until there is an implementation? Thanks.

Reply:
Added :-)